css/chat.css

```diff
@@ -531,7 +529,7 @@ html.no-supports-container.native body,
   bottom: 0;
   left: 0;
   background: #ffffff;
-  z-index: 11;
```

Comment:
@tonytlwu this line had conflicts between master and production. Do you know what the correct value is? Master is 1010 so I assume 1010 is fine.

Reply:
Yes. 1010 is fine.